tests/test_operation.py

```diff
@@ -102,7 +102,8 @@ def h_test(cls):
                 U_high_order = np.array([U] * 3)
                 op.heisenberg_expand(U_high_order, len(op.wires))
 
-        for cls in pennylane.ops.cv.all_ops + pennylane.expval.cv.all_ops:
+        for op in pennylane.ops._cv__ops__ | pennylane.ops._cv__obs__:
```

Comment:
how come you switched over to `|` here?

Reply:
Because the Pauli matrices have the weird property now of being both operations _and_ observables, it made it easier to make these attributes be sets (so that we can use set logic on them).